nemo/collections/tts/models/radtts.py

```diff
@@ -0,0 +1,419 @@
+# Copyright (c) 2022, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# ##########################################################################
+
+# import argparse
+import os
+
+import torch
+from hydra.utils import instantiate
+from omegaconf import MISSING, DictConfig, OmegaConf
+from pytorch_lightning import Trainer
+from pytorch_lightning.loggers import LoggerCollection, TensorBoardLogger
+from torch.optim.lr_scheduler import CosineAnnealingWarmRestarts, ExponentialLR, ReduceLROnPlateau, StepLR
+
+from nemo.collections.tts.helpers.radam import RAdam
+from nemo.collections.tts.losses.radttsloss import AttentionBinarizationLoss, RADTTSLoss
+from nemo.collections.tts.models.base import SpectrogramGenerator
+from nemo.collections.tts.modules.alignment import plot_alignment_to_numpy
+from nemo.collections.tts.torch.tts_tokenizers import BaseTokenizer
+from nemo.core.classes import Exportable
+from nemo.utils import logging
+
+torch.cuda.empty_cache()
```

Comment:
What is this for?

Reply:
I was using it to clear the cache from previous runs. But we don't really need, so removed it now.